site/src/components/WorkspaceActions/WorkspaceActions.tsx

```diff
@@ -34,7 +41,23 @@ export const WorkspaceActions: FC<WorkspaceActionsProps> = ({
     workspace.latest_build,
   )
   const workspaceState = WorkspaceStateEnum[workspaceStatus]
-  const actions = WorkspaceStateActions[workspaceState]
+
+  const canUpdateWorkspace = workspace.outdated && canAcceptJobs(workspaceStatus)
```

Comment:
`canUpdateWorkspace` has been used previously to refer to update permissions on the workspace: https://github.com/coder/coder/blob/main/site/src/pages/WorkspacePage/WorkspacePage.tsx#L37. Maybe we can rename it to `canBeUpdated`?